packages/datadog-plugin-aws-sdk/test/index.spec.js

```diff
@@ -230,6 +230,14 @@ describe('Plugin', () => {
             helpers.baseSpecs(semver, version, agent, getService, operation,
               serviceName, klass, s3Params, key, metadata)
           })
+
+          it('upload works (regression test for #992)', (done) => {
```

Comment:
This should have a more meaningful name, maybe something like "it('should not break uploading')" or even something that explains what it's actually testing. The comment should also be moved to an actual comment.